### PR TITLE
Improve code quality according to Stephanie's comments #159

### PR DESCRIPTION
**Pull request by WU-LUOYU-SERENA**

### CODE REVIEW

File: src/main/java/scene/clue/CheckedClueTrackerBuilder.java [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
-
-    public SearchedClueTracker(Ui ui) {
-        this.ui = ui;
-        this.suspects = new SuspectList(ui);
-        buildSuspectList();
-    }*/
 
+    /** Builds a tracker for already checked clues according
+     *  to a text file that records relevant information.
+     *
```
Comment (arcturusz) [inline src/main/java/scene/clue/CheckedClueTrackerBuilder.java:13]: Can change the Bulids to next line to follow the javadocs format

File: src/main/java/command/ViewCommand.java [hunk 2]
```diff
@@ -18,18 +18,34 @@ public ViewCommand(String args) {
...
     public void execute(Ui ui, Investigation investigation, SceneList sceneList) {
         ui.printViewingCheckedCluesMessage();
+        findSearchedClues(ui, investigation);
+    }
+
+    /**
+     * Finds the already searched clues for suspects specified by the user.
+     * @param ui used to communicate with the user.
+     * @param investigation contains method to query the already searched clues.
+     */
```
Comment (arcturusz) [inline src/main/java/command/ViewCommand.java:41]: Spacing between the description and param

File: src/main/java/scene/clue/CheckedClueTrackerBuilder.java (hunk 1 above)
Comment (WU-LUOYU-SERENA) [reply src/main/java/scene/clue/CheckedClueTrackerBuilder.java]: Thanks for the comments! have changed accordingly

File: src/main/java/command/ViewCommand.java [hunk 3]
```diff
@@ -18,18 +18,35 @@ public ViewCommand(String args) {
...
+        findSearchedClues(ui, investigation);
+    }
+
+    /**
+     * Finds the already searched clues for suspects specified by the user.
+     * 
+     * @param ui used to communicate with the user.
+     * @param investigation contains method to query the already searched clues.
+     */
+    private void findSearchedClues(Ui ui, Investigation investigation) {
```
Comment (arcturusz) [inline src/main/java/command/ViewCommand.java:43]: I think can standardize if want to put spacing to align the description. I think the params description need start with capital letter.

File: src/main/java/scene/clue/CheckedClueTrackerBuilder.java [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
-    public SearchedClueTracker(Ui ui) {
-        this.ui = ui;
-        this.suspects = new SuspectList(ui);
-        buildSuspectList();
-    }*/
 
+    /**
+     *  Builds a tracker for already checked clues according
+     *  to a text file that records relevant information.
+     *
```
Comment (arcturusz) [inline src/main/java/scene/clue/CheckedClueTrackerBuilder.java:14]: I think should only put 1 space after the * before the 'Builds' and 'to'

File: src/main/java/scene/clue/Clue.java [hunk 5]
```diff
@@ -13,6 +13,14 @@ public Clue() {
...
     }
 
+    /**
+     * Construct a Clue object by setting the clue name, clue image and clue
+     * description to those passed in correspondingly.
+     *
+     * @param clueName name of the clue (usually summarises the gist of the clue)
+     * @param image ascii art that gives visual representation of a clue
+     * @param description details about a clue in words
+     */
```
Comment (arcturusz) [inline src/main/java/scene/clue/Clue.java:23]: Params description should end with punctuation if not will fail the checkstyle

File: src/main/java/ui/Ui.java [hunk 6]
```diff
@@ -330,15 +330,31 @@ public void printDeleteAllNoteMessage() {
...
         System.out.println("Invalid index! Please input a number corresponding to one of the notes shown above.");
     }
-    //@@author
+
+    /**
+     * prints the already searched clues relating to a specifc suspect
+     *
+     * @param name name of one of the suspects
+     * @param clues list of already checked clues under that suspect
+     */
```
Comment (arcturusz) [inline src/main/java/ui/Ui.java:349]: I think should start the description of method with a capital letter for this.

**Review by arcturusz — COMMENTED** [review]
I think the params description should start with a capital letter and need to add punctuation at the end of each descriptions else the checkstyle will fail.

**Review by arcturusz — APPROVED** [review]

**Pull request merged**